lib/Controller/PageController.php

```diff
@@ -52,19 +48,17 @@
 use OCP\IUserManager;
 use OCP\Mail\IMailer;
 use OCP\Security\ISecureRandom;
-use OCP\User;
+use OCP\User; //depricated
```

Comment:
We should remove that comment. Perhaps add sth. like `// TODO: Replace with ...`

Reply:
It just seems so... 😁

Reply:
Strange, that there was no error.

Reply:
I updated the code. Hope, everything is fine now.